ambry-commons/src/test/java/com/github/ambry/compression/ZstdCompressionTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2022 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.compression;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ZstdCompressionTest {
```

Comment:
we can merge ZstdCompressionTest and ZstdCompressionFailedTest, just like the LZ4 tests.